src/include/ipc/dai-imx.h

```diff
@@ -20,6 +20,7 @@ struct sof_ipc_dai_esai_params {
 	uint16_t mclk_id;
 	uint32_t mclk_rate; /* MCLK frequency in Hz */
 	uint32_t mclk_direction;
+	uint32_t fsync_rate;
 
```

Comment:
@iuliana-prodan one thing. Lets align this structure with the one in kernel include/sound/sof/dai-imx.h. This looks to be part of the ABI. Actually, looks a little bit broken but we never used so it was fine until now.

Reply:
done.